src/Kiota.Builder/Writers/CSharp/CSharpConventionService.cs

```diff
@@ -13,7 +13,8 @@ public class CSharpConventionService : CommonLanguageConventionService {
         public override string VoidTypeName => "void";
         public override string DocCommentPrefix => "/// ";
         private static readonly HashSet<string> NullableTypes = new(StringComparer.OrdinalIgnoreCase) { "int", "bool", "float", "double", "decimal", "long", "Guid", "DateTimeOffset", "TimeSpan", "Date","Time", "sbyte", "byte" };
-        public static readonly char NullableMarker = '?';
+        public const char NullableMarker = '?';
+        public const string NullableEnableDirective = "NETSTANDARD2_1_OR_GREATER || NET6_0_OR_GREATER";
```

Comment:
```suggestion
        public const string NullableEnableDirective = "NETSTANDARD2_1_OR_GREATER || NETCOREAPP3_1_OR_GREATER";
```

https://github.com/dotnet/designs/blob/main/accepted/2020/or-greater-defines/or-greater-defines.md#design
https://www.meziantou.net/csharp-8-nullable-reference-types.htm

Reply:
Thanks for the heads up. I defaulted to `NET6_0_OR_GREATER` as `NETCOREAPP3_1_OR_GREATER` as I figured we wouldn't expect usage of out of support versions. But will resolve this as its provides more coverage.